frontend/src/Page.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react'
 import { BrowserRouter as Router } from 'react-router-dom'
-
+import { useMediaQuery } from 'react-responsive'
 import Header from './Header'
+import HeaderForMobile from './HeaderForMobile'
 import Menu from './Menu'
 import './style.css'
 import Routing from './Routing'
 
 export default function Page() {
+  const isMobile = useMediaQuery({ query: `(max-width: 760px)` })
   return (
     <Router>
       <header>
         <div className="col-12">
-          <Header />
+          {isMobile ? <HeaderForMobile /> : <Header />}
```

Comment:
Én nem húztam volna be a react-responsive-et csak emiatt, d-none, d-sm-block, d-sm-none classokkal simán meg lehet oldani. Esetleg ha nagyon hasonló a két header, akkor csak egy-két elemet elrejteni megjeleníteni így és a többit ugyanúgy használni.